src/coreComponents/physicsSolvers/contact/SolidMechanicsALMBubbleKernels.hpp

```diff
@@ -330,6 +330,21 @@ class ALMBubbleKernels :
     LvArray::tensorOps::scaledAdd< nBubbleUdof, nUdof >( stack.localAbu, Abu_gauss, -detJ );
     LvArray::tensorOps::scaledAdd< nUdof, nBubbleUdof >( stack.localAub, Aub_gauss, -detJ );
 
+    // Compute the initial stress
```

Comment:
what do you mean by `initialStress` ?

Reply:
If the initial stress is non-zero, as specified for example in the input file `SingleFracCompression_base.xml`, I need to account for the contribution from the bubble functions when integrating the momentum balance equation. Currently, this contribution is computed based on the assumption that the constitutive model is linear elastic.